src/libraries/System.Linq/tests/AggregateByTests.cs

```diff
@@ -93,65 +93,50 @@ public void AggregateBy_SourceThrowsOnCurrent()
             Assert.Throws<InvalidOperationException>(() => enumerator.MoveNext());
         }
 
-        [Theory]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/92387", typeof(PlatformDetection), nameof(PlatformDetection.IsNativeAot))]
-        [MemberData(nameof(AggregateBy_TestData))]
-        public static void AggregateBy_HasExpectedOutput<TSource, TKey, TAccumulate>(IEnumerable<TSource> source, Func<TSource, TKey> keySelector, Func<TKey, TAccumulate> seedSelector, Func<TAccumulate, TSource, TAccumulate> func, IEqualityComparer<TKey>? comparer, IEnumerable<KeyValuePair<TKey, TAccumulate>> expected)
-        {
-            Assert.Equal(expected, source.AggregateBy(keySelector, seedSelector, func, comparer));
-        }
-
-        [Theory]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/92387", typeof(PlatformDetection), nameof(PlatformDetection.IsNativeAot))]
-        [MemberData(nameof(AggregateBy_TestData))]
-        public static void AggregateBy_RunOnce_HasExpectedOutput<TSource, TKey, TAccumulate>(IEnumerable<TSource> source, Func<TSource, TKey> keySelector, Func<TKey, TAccumulate> seedSelector, Func<TAccumulate, TSource, TAccumulate> func, IEqualityComparer<TKey>? comparer, IEnumerable<KeyValuePair<TKey, TAccumulate>> expected)
-        {
-            Assert.Equal(expected, source.RunOnce().AggregateBy(keySelector, seedSelector, func, comparer));
-        }
-
-        public static IEnumerable<object[]> AggregateBy_TestData()
+        [Fact]
```

Comment:
Our usual approach when fixing failures like this is to list missing specializations in the corresponding [default.rd.xml](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Linq/tests/default.rd.xml), which should hopefully no longer be necessary once generic theories are supported in Native AOT I believe @agocke has been working on this space.

Reply:
That's a benefit of theories, yes, and we give up on that benefit frequently for things off the golden path, such as more inputs than the system can handle, places where theories aren't conducive to the kinds of work being done, or places like here where without significant overhaul to how they're implemented they're not compatible with aot. I'd much rather give up on that minor benefit that have this fail in complicated ways when we add one more input.

Reply:
It's possible the work Andy is doing won't even support generic theories. It's not exactly trivial to figure out the type information necessary to source generate the actual generic parameters to call the theory with at compile time.

Reply:
I believe it's possible for it to work in principle, although it might require constraining what generic theories are allowed or adopting a new DSL. In any case, we can adopt this approach for now to unblock testing.